monitoring/README.md

```diff
@@ -92,47 +157,108 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
 a) API Server Monitoring: input into the following text boxes.
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
```

Comment:
At this point the user should be told how to find the right values for "Instance".

Reply:
This turns up a lot:

```console
mspreitz@mjs13 ~ % kubectl --context kind-kubeflex get Endpoints -A | egrep "NAME| wds| vcluster |kubernetes"
NAMESPACE         NAME                                                              ENDPOINTS                                               AGE
default           kubernetes                                                        172.19.0.2:6443                                         97m
its1-system       vcluster                                                          10.244.0.14:8443,10.244.0.14:8443                       94m
wds1-system       wds1                                                              10.244.0.17:9444                                        93m
```

Reply:
Fixed! Using the simpler way provided above. Thanks!